src/lib/integration_test_cloud_engine/constants.ml

```diff
@@ -0,0 +1,13 @@
+let aws_region = "us-west-2"
```

Comment:
Nit: please rename this file to cluster_defaults or infra_defaults.  it's not really "constants" IMO